dist/metacom.js

```diff
@@ -226,9 +214,8 @@ class HttpTransport extends Metacom {
       body: data,
     }).then((res) => {
       const { status } = res;
-      if (status === 200) {
+      if (status === 200 || status === 500) {
```

Comment:
Why just 500, do we need that for 404 for example?

Reply:
I tested and it is correct. There is no need for any additional status handling here. Just use `Metacom.message` it is more then enough.